SPECS/nodejs/nodejs.spec

```diff
@@ -1,25 +1,33 @@
 Summary:        A JavaScript runtime built on Chrome's V8 JavaScript engine.
 Name:           nodejs
-Version:        14.17.5
+Version:        14.18.1
 Release:        1%{?dist}
 License:        BSD and MIT and Public Domain and naist-2003
-Group:          Applications/System
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
+Group:          Applications/System
 URL:            https://github.com/nodejs/node
-Source0:        https://nodejs.org/download/release/v%{version}/node-v%{version}.tar.xz
+# Source0 has a vendored OpenSSL source tree with patented algorithms.
+# Use Source1 to create a clean and reproducible source tarball.
+#Source0:       https://nodejs.org/download/release/v%{version}/node-v%{version}.tar.xz
+Source0:        node-v%{version}-clean.tar.xz
+Source1:        clean-source-tarball.sh
 Patch0:         patch_tls_nodejs14.patch
-
+Patch1:         remove_unsupported_tlsv13_ciphers.patch
 BuildRequires:  coreutils >= 8.22
-BuildRequires:  openssl-devel >= 1.0.1
+BuildRequires:  openssl-devel >= 1.1.1
 BuildRequires:  python3
 BuildRequires:  which
+# ca-certificates is needed to communicate with default npm registry
+Requires:       ca-certificates
```

Comment:
I wouldn't add a `Requires` on any specific certificate bundle anywhere - we don't want to make the system start trusting hundreds of CAs (most of them not required by `npm`) without user's explicit request and/or knowledge. Or to look at this from a different perspective: `npm` will still run without the certificates, may not be able to authenticate the server, but `ca-certificates` is not guaranteed to contain the trusted CAs it needs.

What's probably more important, though, is that `npm` is only one component of `nodejs` and I the user might not be even interested in using it. And if they do, a server authentication error message seems reasonable.

Reply:
Sure thing, this sounds reasonable from a user configurability perspective. Updated to remove the requirement.